src/connector/src/source/pulsar/mod.rs

```diff
@@ -24,7 +24,7 @@ pub use split::*;
 
 pub const PULSAR_CONNECTOR: &str = "pulsar";
 
-#[derive(Clone, Debug, Deserialize)]
+#[derive(Clone, Debug, Deserialize, Default)]
```

Comment:
I think it is better to impl `Default` trait manually for clarity

Reply:
After thinking about it, it is confusing to give this kind of implementation Default, so I removed the Default